index.html

```diff
@@ -2576,8 +2561,8 @@ <h3>The <dfn>NFCScanOptions</dfn> dictionary</h3>
         class="example highlight">
         const options = {
           id: "https://www.w3.org/*",  // any path from the domain is accepted
-          recordType: "json",
-          mediaType: "application/*+json"  // any JSON-based MIME type
+          recordType: "media",
+          mediaType: "application/*json"  // any JSON-based MIME type
```

Comment:
I believe it was a typo. Please tell me if I'm wrong.

Reply:
Yes, please correct these.